### ISSUE TITLE
Skip a file

### ISSUE THREAD

**Issue by zhamid-src** [aljazsim/vs-code-typescript-class-organizer#61]
Is it possible to skip a file from organization?  There are some files that I like to organize and I don't want them to be reorganized automatically (I set the class organizer to run on save). 

Alternatively, if I can skip specific interfaces then that would be helpful. I have database models defined as interfaces. This extension sorts them in alphabetical order which makes it hard to follow. I want to group fields together, but can't figure out how to do that.

**Comment by aljazsim** [comment]
It is  now! From version 1.0.27 any TS file starting with 
```
// tsco:ignore
```

or

```
// <auto-generated />
```

will be ignored when organizing a file!
